lib/tests/test_flows.cpp

```diff
@@ -206,6 +207,11 @@ TEST_CASE("Data Flow : Create/Destroy", "[mxl flows]")
     uint64_t index = mxlTimestampToIndex(&rate, now);
     REQUIRE(index != MXL_UNDEFINED_INDEX);
 
+    /// validate grain and flow structure sizes
+    REQUIRE(sizeof(GrainInfo) == 0x1000);
+    REQUIRE(sizeof(FlowInfo) == 0x1000);
+    REQUIRE(sizeof(mxl::lib::Grain) == 0x2000);
+
```

Comment:
Checking these is a great idea, but in order to avoid pitfalls as the one above we should probably also check for `std::has_unique_object_representations_v<...>` of each type.

There are non-padding related reasons why this trait might not be fulfilled, but as long as those are not a factor we can put it to good use here.